components/Dashboard/Cards/DashboardCard.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Paper, Box, Typography } from '@material-ui/core';
+import { useTheme } from '@material-ui/core/styles';
+
+export default function DashboardCard(props) {
+  const theme = useTheme();
+  const { spaceName, description, variant } = props;
+  return (
+    <Paper className="rounded-xl " elevate={5}>
+      <Box
+        style={{
+          background:
+            variant === 'primary'
+              ? theme.palette.primary.mainGradient
+              : theme.palette.secondary.mainGradient,
+        }}
+        className="flex flex-grow justify-between rounded-r-xl rounded-bl-xl ">
+        <div className="h-32 w-80 grid grid-cols-3">
```

Comment:
Im confused. Are we using Material UI for components and Tailwind for style? I think u can have a `Grid` here right?

Reply:
Very true. I'm not sure why this is showing up in my pr. I didn't write this file lol.